assets/css/main.css

```diff
@@ -0,0 +1,28 @@
+body{
+    margin:0;
+    padding:0;
+}
+.container{
```

Comment:
Jak już definiujesz sobie container, to coś z nim rób.
Nigdy nie wystawiaj pull request z pustą klasą. 

Rozumiem że sobie przygotowałeś na później, ale nie można czegoś takiego mergować do kodu produkcyjnego.
Poza tym na ten moment wystarczyło mu dać width: 100%; i problem rozwiązany.

Reply:
Poprawione